flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/AbstractFlinkService.java

```diff
@@ -326,6 +326,7 @@ protected void cancelJob(
                                 exception);
                     }
                     if (deleteClusterAfterSavepoint) {
+                        LOG.info("Cleaning up deployment after savepoint");
```

Comment:
Maybe we should be more specific as in `stop-with-savepoint` or `savepoing-shutdown`.